src/extension/geofence-server/src/test/java/org/geoserver/geofence/rest/RulesRestControllerTest.java

```diff
@@ -576,7 +580,7 @@ public void testRestControllerPaths() throws Exception {
                 assertEquals("DE_USNG_UTM18", layerDetails.getString("defaultStyle"));
                 assertEquals("Northings >= 100", layerDetails.getString("cqlFilterRead"));
                 assertEquals(
-                        "MULTIPOLYGON (((-180 -90, -180 90, 180 90, 180 -90, -180 -90)))",
+                        "SRID=4326;MULTIPOLYGON (((-180 -90, -180 90, 180 90, 180 -90, -180 -90)))",
```

Comment:
Interesting... this is making the API non backwards compatible though, or am I misunderstanding?

Reply:
Confirmed, it is not backward compatible.
The GeoFence part of the SRID user story did break compatibility, so that GeoFence had to change version from 3.4 to 3.5

Reply:
See also https://github.com/geoserver/geofence/wiki/Compatibility-matrix-(GS-2.17-2.19,-GF-3.4.x-3.5.x)

Reply:
Hum... I guess it's late, but just thinking out loud, wondering if it was really necessary to do... the SRS could have been a separate field, with a default value making it work like before?

Reply:
It may be Good that the whole field become unparsable between different versions: if new data arrives to old authorization logic, the old logic would deal with the geometry as it was 4326, possibly giving access to unauthorized areas, and probably this could be even worse than a runtime error.

Reply:
@etj that's not how GeoServer handle backwards compatibility. GeoServer upgrades must be transparent if at all possible, an old client should keep on working un-modified against a newer version of GeoServer.

If someone tries to use a newer client against an old version of the server, that's their problem, we're not in control of clients.

That said, in this case I'm guessing damage is done already. Please keep this in mind and avoid backwards incompatibilities, if at all  possible, in the future.